book/src/user/mining-testnet-s-nomp.md

```diff
@@ -73,11 +73,12 @@ These fixes disable mining pool operator payments and miner payments: they just
 
     </details>
 
-2. [Build](https://github.com/ZcashFoundation/zebra#build-instructions) and [Run Zebra](https://zebra.zfnd.org/user/run.html) with the `getblocktemplate-rpcs` feature:
+2. [Run Zebra](https://zebra.zfnd.org/user/run.html) with the config you created:
     ```sh
-    cargo run --release --features "getblocktemplate-rpcs" --bin zebrad -- -c zebrad.toml
+    zebrad -c zebrad.toml
     ```
-3. Wait a few hours for Zebra to sync to the testnet tip (on mainnet this takes 2-3 days)
+3. Wait for Zebra to sync to the testnet tip. This takes 2-3 days on Mainnet at
+   the time of writing this manual.
```

Comment:
I know the testnet sync time if that helps?

Also I usually put dates so we know when things should be updated. (And users know how outdated they are.)

```suggestion
3. Wait for Zebra to sync to the testnet tip. This takes 8-12 hours on testnet (or 2-3 days on mainnet) as of October 2023.
```

Reply:
Yea, that's much better. Thank you.